rabbitmq/check.py

```diff
@@ -353,7 +353,8 @@ def get_connections_stat(self, instance, base_url, object_type, vhosts, custom_t
         for conn in data:
             if conn['vhost'] in vhosts:
                 stats[conn['vhost']] += 1
-                connection_states[conn['state']] += 1
+                if conn.get('state'):
```

Comment:
Instead of skipping the connection we could create an `unknown` status.

I think this would do the trick
```
connection_states[conn.get("state", "unknown")] += 1
```

Reply:
https://cl.ly/0z3w310T2n1d
I reckon there is no 'state' if the connection is of type='direct' and/or protocol='Direct.*' as opposed to type='network' and protocol='AMQP.*". 
Not to sure whether it would be a good idea to send in a metric with tag rabbitmq_conn_state:unknown. 
What about -
`connection_states[conn.get("state", "direct")] += 1`

Reply:
Thanks for the information !